datalake/dlfile.py

```diff
@@ -122,7 +122,7 @@ def from_bundle(cls, bundle_filename):
 
         '''
         cls._validate_bundle(bundle_filename)
-        b = tarfile.open(bundle_filename, 'r:')
+        b = tarfile.open(bundle_filename, 'r:gz')
```

Comment:
let's keep this backward-compatible. Maybe check to see if the file is compressed like this: http://stackoverflow.com/questions/3703276/how-to-tell-if-a-file-is-gzip-compressed